test/integration/BLACKBOX_README.md

```diff
@@ -0,0 +1,398 @@
+# TEAL Blackbox Toolkit: Program Reporting and Testing via Dry Runs
```

Comment:
I see "TEAL Blackbox Toolkit" and "TEAL Blackbox Testing" : are these two different concepts or the same? (is there more to the toolkit than testing?)